tool/src/webapp/js/clog_utils.js

```diff
@@ -377,12 +378,11 @@ clog.utils = {
 
         return false;
     },
-    findPost: function (postId, callback) {
-        
-        if (!clog.currentPosts) {
+    findPost: function (postId, callback, fromSamepage) {
 
-            jQuery.ajax( {
-                url: "/direct/clog-post/" + postId + ".json",
+        if (!clog.currentPosts || fromSamepage) {
+            jQuery.ajax({
```

Comment:
The indentations have been fixed.
I could not find anything that needed to be changed with the indentations on lines 380-384